### PR TITLE
Info about startup parameters

### PR DESCRIPTION
**Pull request by bmmpxf**

### CODE REVIEW

File: docs/usermanual/source/sysadmin/startup/index.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
+.. _sysadmin.startup:
+
+How to add startup parameters for GeoServer
+===========================================
+
+Many configuration settings for GeoServer must be set during the startup process. Startup options have the prefix ``-D`` and are of the form ``OPTION=value``.
```
Comment (jodygarnett) [inline docs/usermanual/source/sysadmin/startup/index.rst]: You could swing this as a positive (also they are called system properties, not all application servers configure these settings using -D command line options).

GeoServer allows global configuration settings to be provided as Java System Properties for use during the startup process. 

Java system properties can be supplied on the command line using  ``-D`` (and are of the form ``-D property=value``).

File: docs/usermanual/source/sysadmin/startup/index.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+This section shows how to set startup-level configuration settings.
```
Comment (jodygarnett) [inline docs/usermanual/source/sysadmin/startup/index.rst]: This section shows how to set the Java System Properties used during startup.

File: docs/usermanual/source/sysadmin/startup/index.rst [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+Jetty
+-----
+
+OpenGeo Suite for Windows and OS X both use the Jetty application server. Startup flags are all contained in a file called :file:`start.ini`.
+
+On Windows, this file is contained in :file:`jetty\\start.ini` inside the root of the installation directory. For example, :file:`C:\\Program Files\\Boundless\\OpenGeo\\jetty\\start.ini`.
+
+To access the :file:`start.ini` file on OS X, click :guilabel:`Open Webapps` from the GeoServer menu. The file is one level up in the directory tree.
+
+.. figure:: img/startini.png
```
Comment (jodygarnett) [inline docs/usermanual/source/sysadmin/startup/index.rst]: The diagram is nice and clear, wonder if we could use an "inline" and grab the real ini file link (so this picture won't go out of date).

This file contains a mix of JVM parameters (for memory etc...) and system properties.

File: docs/usermanual/source/sysadmin/startup/index.rst [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+   A sample start.ini file
+
+There is a section titled :guilabel:`other geoserver options` where startup options can go, but the specific order is not important. Each option must be on its own line.
```
Comment (jodygarnett) [inline docs/usermanual/source/sysadmin/startup/index.rst]: java system properties

File: docs/usermanual/source/sysadmin/startup/index.rst [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+#. Save and close the file.
+
+#. Restart GeoServer.
+
+Tomcat
+------
+
+When using OpenGeo Suite with Tomcat, we recommend you add the startup options to the :file:`setenv.sh` (Linux) or :file:`setenv.bat` (Windows) script file. This file is typically found in the :file:`bin` directory of the Tomcat installation, for example :file:`/usr/share/tomcat7/bin`.
+
+.. note:: If this file doesn't exist, you can create it in the same directory as the other Tomcat startup scripts such as :file:`catalina.sh` (Linux) or :file:`catalina.bat` (Windows).
```
Comment (jodygarnett) [inline docs/usermanual/source/sysadmin/startup/index.rst]: Thanks for this, it is a common mistake to edit the catalina.bat directly ...

**Comment by jodygarnett** [comment]
A few comments above, but it looks good.

**Comment by bmmpxf** [comment]
Thanks! feedback was very helpful. I made most of the changes. Stuff like linking to the INI is nice, but not really necessary, and personally I like the visual contrast of having an image in the middle of the page. If start.ini changes a bunch and looks really off, happy to revisit.

**Pull request merged**